lib/ASRouter.jsm

```diff
@@ -664,6 +670,61 @@ class _ASRouter {
     });
   }
 
+  hasGroupsEnabled(groups = []) {
+    return this.state.groups
+      .filter(({ id }) => groups.includes(id))
+      .every(({ enabled }) => enabled);
```

Comment:
Do we want this to return `true` if a message is part of no groups (I guess that's not really possible... but maybe if there's a failure case. Probably more efficient to do `every({ id, enabled }) => !groups.includes(id) || enabled`

Reply:
Using `!groups.includes(id)` predicate for `.every` We‘ll also cover the case for `[]`.